### PR TITLE
[ZEE-6187] Add LongText property

### PR DESCRIPTION
**Pull request by martin-guerre**
JIRA issue: https://zeenea.atlassian.net/browse/ZEE-6187

##### PR summary:

- Add LongText property

##### Checklist for Developer:

- [x] Summary part has been documented
- [x] Reviewers have been requested
- [ ] Reviews & comments have been taken into consideration
- [ ] Commits have been reworded and history cleaned

##### Checklist for Reviewer

- [ ] Code has been reviewed, commented and changes have been requested
- [ ] Reviews & comments have been taken into consideration

### CODE REVIEW

**Review by flalles — APPROVED** [review]

File: src/main/java/zeenea/connector/property/PropertyType.java [hunk 1]
```diff
@@ -2,9 +2,20 @@
...
+   */
   STRING,
 
+  /**
+   * Represents a long text property type.
+   *
+   * <p>Long text property value is not indexed and searchable in data catalog.
+   */
+  LONG_TEXT,
+
```
Comment (arthurtemple) [inline src/main/java/zeenea/connector/property/PropertyType.java:18]: :+1:

File: src/main/java/zeenea/connector/property/LongTextPropertyValue.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import java.util.Objects;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Represents a property value of long text type. This class is immutable and implements the
+ * PropertyValue interface.
+ *
+ * <p>Long text property value is not indexed and searchable in data catalog.
+ */
+public final class LongTextPropertyValue implements PropertyValue {
```
Comment (arthurtemple) [inline src/main/java/zeenea/connector/property/LongTextPropertyValue.java:12]: Il remplacer `StringPropertyValue` par `LongTextPropertyValue` partout dans ce fichier.

Comment (martin-guerre) [reply src/main/java/zeenea/connector/property/LongTextPropertyValue.java:12]: Ah bien vu ! C'est corrigé

File: src/main/java/zeenea/connector/property/LongTextPropertyValue.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  }
+
+  /**
+   * Returns the string representation of this LongTextPropertyValue.
+   *
+   * @return the string representation of this LongTextPropertyValue
+   */
+  @Override
+  public String toString() {
+    return "StringPropertyValue{" + "value='" + value + "'}";
```
Comment (franck-wlodarezack) [inline src/main/java/zeenea/connector/property/LongTextPropertyValue.java:66]: Si je ne dis pas de bêtise :
```suggestion
    return "LongTextPropertyValue{" + "value='" + value + "'}";
```

**Review by franck-wlodarezack — APPROVED** [review]

File: src/main/java/zeenea/connector/property/LongTextPropertyValue.java (hunk 3 above)
Comment (martin-guerre) [reply src/main/java/zeenea/connector/property/LongTextPropertyValue.java:66]: Très bien vu aussi, c'est corrigé (et normalement il n'y en a plus !)

**Pull request merged**